enterprise/models.py

```diff
@@ -4369,6 +4376,8 @@ def get_all_learners(self, user_query=None, sort_by=None, desc_order=False, fetc
                 'recent_action': lambda t: t.recent_action,
             }
             members = sorted(members, key=lambda_keys.get(sort_by), reverse=desc_order)
+        if pending_users_only:
```

Comment:
I don't think this can be tact onto the end of the other work being done - say you have an implicitly defined group and pass the `pending_users_only` bool, you would theoretically receive 0 records. Instead I think `pending_users_only` should be passed into both `_get_explicit_group_members` and `_get_implicit_group_members` above and handled by the methods. For the implicit fetch, all you have to do is skip the steps [here](https://github.com/openedx/edx-enterprise/pull/2049/files#diff-d950991245f878f58adc157895d6431af4385d522ab994aa25726e9abc2b434dL4307-L4317) that involve the customer_users if the `pending_users_only` value is True

Reply:
(and instead have `customer_users` be an empty array (`[]`)